### ISSUE TITLE
Typeahead on main page fails accessibility audit in chrome

### ISSUE THREAD

**Issue by Falkirks** [pronouns/main#8]
Fails due to 
```
[role]s that require child [role]s contain them.
Some ARIA parent roles require specific roles on their children to perform their accessibility function.
```

**Comment by Falkirks** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [1.3.1 Info and Relationships](https://trello.com/c/l0dahbPq/11-131-info-and-relationships)
